components/base/sensorcontrolled/movestraight.go

```diff
@@ -54,6 +55,16 @@ func (sb *sensorBase) MoveStraight(
 		}
 	}
 
+	// if loop has been tuned but the values haven't been added to the config, error with tuned values
+	if (sb.configPIDVals[0].NeedsAutoTuning() && !(*sb.tunedVals)[0].NeedsAutoTuning()) ||
+		(sb.configPIDVals[1].NeedsAutoTuning() && !(*sb.tunedVals)[1].NeedsAutoTuning()) {
+		return fmt.Errorf("%v has been tuned, copy these control parameters into the config to enable movement: "+
+			"\"control_parameters\": [{\"p\": %v, \"i\": %v, \"d\": %v, \"type\": \"linear_velocity\"}, "+
+			"{\"p\": %v, \"i\": %v, \"d\": %v, \"type\": \"angular_velocity\"}]",
+			sb.Name().ShortName(), (*sb.tunedVals)[0].P, (*sb.tunedVals)[0].I, (*sb.tunedVals)[0].D,
+			(*sb.tunedVals)[1].P, (*sb.tunedVals)[1].I, (*sb.tunedVals)[1].D)
+	}
+
```

Comment:
lets turn this message into a function that takes in the list of PID values. could be part of the controls package so it can be used for the motor and base, but just leaving it in the sensorcontrolled package is fine too. 

[q] will `tunedVals` be populated for both `linear_velocity` and `angular_velocity` if only one of the signals gets tuned?

Reply:
if one of them isn't tuned, the `tunedVals` will be all 0. I think that's probs fine because that means a user has values they want already in the config. I thought about setting `tunedVals` to the config PID Vals if they were non-zero, but I didn't because those aren't _techicnally_ the tuned values

Reply:
like if only angular is tuned, `tunedVals` would look like `{{P: 0, I: 0, D, 0}, {P: tunedVals.P, I: tunedVals.I, D: tunedVals.D}}`

Reply:
we spoke about this offline, and are now only displaying values that were actually tuned & not the zeroes